clinica/iotools/converters/adni_to_bids/adni_modalities/adni_fdg_pet.py

```diff
@@ -18,8 +18,8 @@ class ADNIPreprocessingStep(Enum):
     STEP1 = "Co-registered Dynamic"
     STEP2 = "Co-registered, Averaged"
     STEP3 = "Coreg, Avg, Standardized Image and Voxel Size"
-    STEP4_8MM = "Coreg, Avg, Std Img and Voxel Size, Uniform Resolution"
-    STEP4_6MM = "Coreg, Avg, Std Img and Voxel Size, Uniform 6mm Res"
+    STEP4_8MM = "Coreg, Avg, Std Img and Vox Siz, Uniform Resolution"   # noqa
```

Comment:
Looks like I was wrong...
You can try adding "Vox" and "Siz" here:

https://github.com/aramis-lab/clinica/blob/31543c22118914e56165c28fd525c4fe5ded9987/pyproject.toml#L113